doc/tutorials/QGAN.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>quanum -&gt; quantum</p><p><br></p><p>should-&gt; may ? (being more cautious)</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I agree</p><p><br></p>